packages/mask/src/components/InjectedComponents/ProfileCard/ProfileBar.tsx

```diff
@@ -127,7 +129,7 @@ export const ProfileBar = memo<ProfileBarProps>(
                         <Link
```

Comment:
The `Link` will ref to an unexpected URL if `addressLink` is absent.

Reply:
Updated. Now check it before using.